SalesforceFoundation/HEDAP
# Duplicate Department Affiliations are no longer inserted

# Critical Changes

# Changes
- If within a single transaction (via Data Loader, for example) you insert multiple Course Connection or Course Offering records that relate to the same faculty Contact and department, duplicate Affiliation records are no longer created.

# Issues Closed
Fixes #651 

# New Metadata

# Deleted Metadata

# Testing Notes

1. Ensure that Course Connections is enabled in HEDA Settings.
2. Create two Course Connections at the same time (through data loader, etc) for the same Faculty & Department - this should not create duplicate affiliations.
3. Create or update two Course Offerings at the same time (through data loader, etc) for the same Faculty & Department - this should not create duplicate affiliations.
4. Disable Course Connections and repeat step 3.
